README.md

```diff
@@ -1,36 +1,314 @@
+# Build sparrow config
+This section describes how to write a config file for Sparrow.
+If you wish to use just one of the following services, do not include the other in the config.
 
-**TODO: Check API diffs between these articles:** 
+```elixir
+config :sparrow, config: [fcm: [...], apns: [...]]
+```
 
-[link](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification)
-and
-[link](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/PayloadKeyReference.html#//apple_ref/doc/uid/TP40008194-CH17-SW1)
+## Config examples
 
-## Installation
+FCM only:
 
-[![Build Status](https://travis-ci.com/esl/sparrow.svg?branch=master)](https://travis-ci.com/esl/sparrow)
-[![Coverage Status](https://coveralls.io/repos/github/esl/sparrow/badge.svg)](https://coveralls.io/github/esl/sparrow)
+```elixir
+config :sparrow, config: [
+    fcm: [
+        [
+            path_to_json: "path/to/fcm_token.json"
+        ]
+    ]
+]
+```
+
+APNS only:
+
+```elixir
+config :sparrow, config: [
+    apns: [
+        dev:
+        [
+            [
+                auth_type: :certificate_based,
+                cert: "path/to/apns/cert.pem",
+                key: "path/to/apns/key.pem"
+            ]
+        ],
+        prod:
+        [
+            [ 
+                auth_type: :token_based,
+                token_id: :some_atom_id,
+                tags: [:production, :token_based_auth]
+            ],
+            [ 
+                auth_type: :token_based,
+                token_id: :other_atom_id,
+                tags: [:other_production_pool, :token_based_auth]
+            ]
+        ],
+        tokens:
+        [
+            [
+                token_id: :some_atom_id,
+                key_id: "FAKE_KEY_ID",
+                team_id: "FAKE_TEAM_ID",
+                p8_file_path: "path/to/file/token.p8"
+            ],
+             [
+                token_id: :other_atom_id,
+                key_id: "OTHER_FAKE_KEY_ID",
+                team_id: "OTHER_FAKE_TEAM_ID",
+                p8_file_path: "path/to/file/other/token.p8"
+            ]
+        ]
+    ]
+]
+```
+
+Both FCM  and APNS:
+
+```elixir
+config :sparrow, config: [
+    fcm: [
+        [
+            path_to_json: "path/to/fcm_token.json"
+        ]
+    ],
+    apns: [
+        dev:
+        [
+            [
+                auth_type: :certificate_based,
+                cert: "path/to/apns/cert.pem",
+                key: "path/to/apns/key.pem"
+            ]
+        ]
+    ]
+]
+```
+
+## Config options
+
+### FCM specific
+- `:path_to_json` - is a path to a json file provided by FCM
+
+### APNS specific
+- `:auth_type` - defines the authentication type, the allowed values are: `:token_based`, `:certificate_based`
+
+    1. `:token_based` requires setting:
+    - `:token_id` - is a unique atom referring to a token with the same `:token_id` in config
```

Comment:
You probably want to indent those a bit more - see the rendered version

Reply:
good catch